src/server/messages.ts

```diff
@@ -17,3 +17,8 @@ export async function lemmatise(filename: string, project: string, text: string)
     const result = await client.executeCommand("lem");
     return result;
 }
+
+export async function check() {
+    const client = new SOAPClient("x.atf", "test", "&X600166 = test\n#project: test")
+    await client.executeCommand("atf");
+}
```

Comment:
Just to understand, this is executing a simple command on the server and waiting for the answer, to make sure the communication is established before proceeding with the other tests?  If so, that sounds good to me!